app/lib/vendor_api/changes/notes.rb

```diff
@@ -0,0 +1,11 @@
+module VendorAPI
+  module Changes
+    class Notes < VersionChange
+      description 'Presents notes for an application'
+
+      action ApplicationsController, %i[index show]
```

Comment:
[Multiple actions are not supported](https://teacher-services-tech-docs.london.cloudapps.digital/services/apply-for-teacher-training/versioning_vendor_api.html#adding-a-new-endpoint) so you should introduce each endpoint in a separate version change class.